### PR TITLE
Support Ukraine

### PR DESCRIPTION
**Pull request by zverbeta**
Support Ukraine

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by dylhunn** [comment]
Please see this link for more information about Google's response: https://blog.google/inside-google/company-announcements/helping-ukraine/

**Comment by mgechev** [comment]
As @dylhunn mentioned Google supports Ukraine. We'll be happy to do our part of this by adding a banner to angular.io.

@zverbeta, could you change the URL to point to https://www.google.org/ukraine-relief/? Additionally, please make sure you're following the contribution guide. You will have to update the commit message.

Thanks!

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by mgechev** [comment]
@zverbeta would you update the styles of the banner a little? I like the design React uses for their banner. Once we change the styles I will review and we can merge.

**Comment by zverbeta** [comment]
@mgechev I updated styles, ✔️please

File: aio/src/app/app.component.html [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
   <mat-toolbar-row class="notification-container">
-    <aio-notification notificationId="survey-january-2022" expirationDate="2022-02-01" [dismissOnContentClick]="true" (dismissed)="notificationDismissed()">
-      <a href="https://stateofjs.com">
-        <mat-icon class="icon" svgIcon="insert_comment" aria-label="Announcement"></mat-icon>
-        <span class="message">Share your experience with Angular in <b>The State of JavaScript</b></span>
-        <span class="action-button">Go to survey</span>
+    <aio-notification notificationId="war-ukraine" expirationDate="null" [dismissOnContentClick]="false" (dismissed)="notificationDismissed()">
+      <a class="link" target="_blank" rel="noopener" href="https://www.google.org/ukraine-relief/">
+        <span>Support Ukraine &nbsp;</span>
+        <div class="icon" aria-label="Ukraine flag">🇺🇦</div>
```
Comment (SchnWalter) [inline aio/src/app/app.component.html:16]: An SVG is better for systems don't not have a color emoji font installed. IIRC, openSUSE doesn't have one installed by default, and there might be other OSes.

The [flag of Ukraine](https://en.wikipedia.org/wiki/Flag_of_Ukraine) is half `#0057b7` and `#ffd700` with an aspect ratio of 3 by 2.

```svg
<svg height="24" width="24" version="1.1" viewBox="0 0 240 240" xmlns="http://www.w3.org/2000/svg">
  <rect fill="#0057b7" height="72" width="240" x="0" y="48"/>
  <rect fill="#ffd700" height="72" width="240" x="0" y="120"/>
</svg>
```

**Comment by mgechev** [comment]
Can we also address this comment? Thank you for the contribution and the patience :) https://github.com/angular/angular/pull/45376#discussion_r835532169

**Comment by zverbeta** [comment]
@mgechev done.

**Review by mgechev — APPROVED** [review]

**Comment by mgechev** [comment]
@zverbeta approved! ✅ 

There's a merge conflict that you'll have to resolve before we're able to merge.

**Comment by dylhunn** [comment]
@zverbeta Happy to merge this, please tag me once the conflict is resolved and I will merge it.

**Comment by zverbeta** [comment]
@dylhunn I resolved merge conflicts, but test_aio_local exceeded size limit. Could you suggest any ideas why that happend

**Comment by dylhunn** [comment]
@zverbeta I think that's because we are adding new code, you can open `goldens/size-tracking/aio-payloads.json` and update `main` to be 450969

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: size-tracking

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 8e09445610701a7f6670277004e8a4a8585a2060.

**Pull request closed without merging**